### ISSUE TITLE
Allow temporarily sliding away the side panel, without losing the object of our query / URL

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/openstreetmap-website#3303]
Here if the user clicks the light grey X,
![20210827T011441](https://user-images.githubusercontent.com/1230959/131006745-3d09aa8a-dbc7-40b5-9650-ae1055aa5957.jpg)
1. The sidepanel closes, allowing the user to see more of the map.
2. Alas, the orange item in question is also lost.

Therefore, please allow temporarily sliding away the side panel, without losing the object of our query / URL.

**Comment by AntonKhorev** [comment]
Do you also want the ability open the sidebar back and if so then how?

**Comment by jidanni** [comment]
It's quite standard on Google and Bing Maps:
![20221128T022336](https://user-images.githubusercontent.com/1230959/204228946-b6663a3d-2181-46f1-851d-cb1937d0bdf8.jpg)
![20221128T022720](https://user-images.githubusercontent.com/1230959/204229633-1eff3917-364f-4dd1-ac4e-05736d795bcf.jpg)
An X at the top, and an arrow at the right.

**Comment by AntonKhorev** [comment]
"x" doesn't close the panel temporarily on Google maps, "<" on the side does. So you don't want "x" to temporarily close the sidebar, you want an additional button for that, right?

**Comment by jidanni** [comment]
No, I want the X to do the same thing it does currently on all three systems mentioned: zapping the search entirely. 

Therefore the only **new** thing I want is that little arrow that just slides the sidebar **mostly** out of the way.
